src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -347,7 +347,7 @@ The following workflows contain a search space containing a set of possible neur
 
 The following workflows represent python templates that can be used to implement a generic machine learning task.
 
-*PyTorch_Task:* is a simple PyTorch task template.
+*Python_Task:* is a simple PyTorch task template.
```

Comment:
I think it is PyTorch_Task, this template is present in the deep-learning-workflows bucket

Reply:
but maybe your repository is not uptodate versus master ?

Reply:
Let me check that. In any case I will use a new branch/PR to avoid squashing those 14 commits so that it will be cleaner